odoo/http.py

```diff
@@ -1184,6 +1184,9 @@ def session_gc(session_store):
 mimetypes.add_type('application/x-font-ttf', '.ttf')
 # Add potentially missing (detected on windows) svg mime types
 mimetypes.add_type('image/svg+xml', '.svg')
+# this one can be present on windows with the value 'text/plain' which breaks
+# loading js files from an addon's static folder
+mimetypes.add_type('application/javascript', '.js')
```

Comment:
Shouldn't it be "text/javascript" as per the RFC9239 ? https://www.rfc-editor.org/rfc/rfc9239#name-text-javascript